controllers/etcd_controller.go

```diff
@@ -428,6 +428,28 @@ func (r *EtcdReconciler) reconcileConfigMaps(ctx context.Context, logger logr.Lo
 		return nil, err
 	}
 
+	if len(filteredCMs) == 0 {
+		// This block is needed for backward compatibility when etcd-druid is downgraded from v0.9.0 to v0.8.2
+		// etcd-druid v0.9.0 adds alternate labels to the configmap that v0.8.2 does not recognise and this can cause reconciliation to get stuck
```

Comment:
```suggestion
		// etcd-druid v0.9.x adds alternate labels to the configmap that v0.8.x does not recognise and this can cause reconciliation to get stuck
```